docs/units/quantity.rst

```diff
@@ -79,6 +79,49 @@ convert the |quantity| to base S.I. or c.g.s units:
     >>> q.cgs
     <Quantity 240.0 cm / s>
 
+Plotting quantities
+-------------------
+
+|quantity| objects can be conveniently plotted using matplotlib.  This
+feature needs to be explicitly turned on:
+
+.. doctest-requires:: matplotlib
+
+    >>> from astropy.visualization import quantity_support
+    >>> quantity_support()
```

Comment:
Yes, it is mentioned within docstring of `quantity_support()` but not very obvious otherwise.